tests/Jellyfin.Networking.Tests/NetworkParseTests.cs

```diff
@@ -514,5 +514,45 @@ public void TestBindInterfaceOverrides(string source, string lan, string bindAdd
 
             Assert.Equal(intf, result);
         }
+
+        [Theory]
+        [InlineData("185.10.10.10,200.200.200.200", "79.2.3.4", true)]
+        [InlineData("185.10.10.10", "185.10.10.10", false)]
+        [InlineData("", "100.100.100.100", false)]
+
+        public void HasRemoteAccess_GivenWhitelist_AllowsOnlyIpsInWhitelist(string addresses, string remoteIp, bool denied)
+        {
+            // Comma separated list of IP addresses or IP/netmask entries for networks that will be allowed to connect remotely.
+            // If left blank, all remote addresses will be allowed.
+            var conf = new NetworkConfiguration()
+            {
+                EnableIPV4 = true,
+                RemoteIPFilter = addresses.Split(","),
```

Comment:
```suggestion
                RemoteIPFilter = addresses.Split(','),
```
It's only in the tests, but still

Reply:
done all 30